ivy/functional/ivy/gradients.py

```diff
@@ -279,16 +277,16 @@ def _process_func_ret_and_grads(func_ret, grads, retain_grads):
 def _variable(x):
     x = ivy.to_native(x, nested=True)
     ret = ivy.nested_map(
-        x, current_backend(x).variable, include_derived=True, shallow=False
+        current_backend(x).variable, x, include_derived=True, shallow=False
     )
     return ivy.nested_map(ret, ivy.to_ivy, include_derived=True)
```

Comment:
could you please change the function call here as well? Rest seems good to merge, thanks @mattbarrett98 😄